catroid/src/main/res/values/strings.xml

```diff
@@ -1577,5 +1577,6 @@
     <string name="crash_dialog_always_send">Always send</string>
     <string name="crash_dialog_send">Send</string>
     <string name="plus" translatable="false">+</string>
+    <string name="com.crashlytics.android.build_id">RANDOM_UUID</string>
```

Comment:
Ah, k. Please add a comment to this line so people know why this is needed.

Oh, and add `translatable="false"` as well.
```xml
<string name="com.crashlytics.android.build_id" translatable="false">RANDOM_UUID</string>
```